doc/changes/unreleased.md

```diff
@@ -3,3 +3,4 @@
 ## Refactorings
 
  - #125: Updated exaslct and added check for password leak
+ - #127: Reformatted code with latest PTB
```

Comment:
Think in many of the files that the imports could be reduced then. Unfortunately, the tools we have in the PTB don't yet include removing unused imports, like `from Typing import Dict`